src/hotspot/cpu/ppc/gc/z/zAddress_ppc.cpp

```diff
@@ -91,10 +91,14 @@ static size_t probe_valid_max_address_bit() {
 size_t ZPlatformAddressOffsetBits() {
   static const size_t valid_max_address_offset_bits = probe_valid_max_address_bit() + 1;
   const size_t max_address_offset_bits = valid_max_address_offset_bits - 3;
+#ifdef ADDRESS_SANITIZER
+  return MIN2(valid_max_address_offset_bits, (size_t)44);
```

Comment:
I think this PR is ok, but please add a comment like "The max supported value is 44 (because of other internal data structures).".